packages/view/backend/sirius-components-view-form/src/main/resources/model/form.ecore

```diff
@@ -80,6 +80,10 @@
         eType="ecore:EDataType ../../../../../sirius-components-view/src/main/resources/model/view.ecore#//Length"
         defaultValueLiteral="250"/>
   </eClassifiers>
+  <eClassifiers xsi:type="ecore:EClass" name="SplitButtonDescription" eSuperTypes="#//WidgetDescription">
+    <eStructuralFeatures xsi:type="ecore:EReference" name="actions" upperBound="-1"
+        eType="#//ButtonDescription" containment="true"/>
```

Comment:
Why there is no SplitButtonDescriptionStyle?

Reply:
The style of the button is the style of the current action

Reply:
Yes but the "dropdown part" is always the main color theme. Do we really want that? May be, may be not, please be sure about that.

Reply:
You're right, I forgot to add the style to the dropdown button.